client/src/main/webjar/program-details/program-details.route.js

```diff
@@ -0,0 +1,31 @@
+(function (angular) {
+    "use strict";
+
+    var module = angular.module("healthBam.programDetails");
+
+    /**
+     * Registers the programDetails view's URL, template, and controller.
+     * @param $stateProvider
+     */
+    function registerProgramDetailsRoute(
+        $stateProvider
+    ) {
+        $stateProvider.state(
+            "programDetails",
+            {
+                url: "/program-details/{programId:int}",
+                component: "healthBamProgramDetails",
+                parent: "main"
+            }
+        );
```

Comment:
Creates the programDetails page, at the URL specified, as a child of the main parent page (with toolbar, sidenav), and specifies to load the new component at this page.